ReShade/Shaders/Otis/MouseOverlay.fx

```diff
@@ -0,0 +1,33 @@
+///////////////////////////////////////////////////////////////////
+// This effects simply shows the mouse coordinates on the screen. It can be used in combination
+// of effects which use the mousecoordinates supplied by ReShade so you can better see what
+// you're doing.
+///////////////////////////////////////////////////////////////////
+
+#include EFFECT_CONFIG(Otis)
+#include "Common.fx"
+
+#if USE_MOUSEOVERLAY
+
+#pragma message "MouseOverlay by Otis\n"
+
+namespace Otis
+{
+	float4 PS_MouseCoordOverlay(float4 vpos:SV_Position, float2 texcoord: TEXCOORD) :SV_Target
+	{
+		float2 focusPoint = ReShade::MouseCoords * ReShade::PixelSize;
+		if(abs(focusPoint.x - texcoord.x) < MOL_CursorSize && abs(focusPoint.y - texcoord.y) < MOL_CursorSize)
```

Comment:
Found it. The values returned from the calculation coming out of abs are of course in screenspace coords. A multiplication with PixelSize is requires. 
```c
return all((abs(ReShade::MouseCoords-vpos.xy) * ReShade::PixelSize) < MOL_CursorSize) ? float4(MOL_CursorColor, 1.0) : tex2D(ReShade::BackBuffer, texcoord);
```
Not sure if this is much faster than what I had though ;)